### ISSUE TITLE
Results displayed for files that should be skipped based on `.bentoignore` 

### ISSUE THREAD

**Issue by dlukeomalley** [returntocorp/bento#195]
## Overview
My `.bentoignore` doesn't seem to be respected for pyatv.

## Current Behavior
Running `bento check` on a specific directory that includes ignored files shows results for files that should be ignored based on `.gitignore`.

```shell
error: r2c.flake8.F401
   --> pyatv/mrp/protobuf/__init__.py:43:1
      |
   43 |   from pyatv.mrp.protobuf.CryptoPairingMessage_pb2 import CryptoPairingMessage
      |
    = note: 'pyatv.mrp.protobuf.CryptoPairingMessage_pb2.CryptoPairingMessage' imported but unused
```

## Steps to Reproduce

1. Add `pyatv/mrp/protobuf/` or `protobuf/`
2.`bento check pyatv/*` 

## Additional Context
Bento version (run `bento --version`): 0.6.1
Project: https://github.com/postlund/pyatv

pyatv also failed to `bento init` initially and I've had to manually specify directories manually per step #2.

**Comment by DrewDennison** [comment]
Thanks for the bug report. We fixed several `.bentoignore` escaping issues in 0.6 that will be released shortly. Please try it out and let us know if that works for you

**Comment by DrewDennison** [comment]
@dlukeomalley wasn't able to reproduce with the details above but 0.6.2 has several fixes for `.bentoignore` escape patterns specifically around `#` and `\` in the line. Can you provide more details or close based on those fixes?

**Comment by nbrahms** [comment]
This will be fixed in 0.8 ❤️
